online-inference/fastertransformer/sleeper-pod.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Do we need this pod for releasing the example? I typically only do this for dev.

Reply:
Ok, this sounds good.